easybuild/easyconfigs/c/CMake/CMake-3.6.2-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.2'
```

Comment:
@akesandgren you can remove this easyconfig, no longer used?

Either that, or sync it with existing CMake easyconfigs w.r.t. use of `-DCMAKE_PREFIX_PATH=$EBROOTNCURSES`, including `ccmake` in sanity check & style fixes (check with `--check-style`)